adobe_vipm/flows/sync.py

```diff
@@ -1118,3 +1120,20 @@ def sync_all_agreements(mpt_client: MPTClient, *, dry_run: bool) -> None:
     agreements = get_all_agreements(mpt_client)
     for agreement in agreements:
         sync_agreement(mpt_client, agreement, dry_run=dry_run, sync_prices=False)
+
+
+def get_one_time_skus(mpt_client: MPTClient, product_id: str) -> tuple[str]:
```

Comment:
I think it can be quite costly to fetch all the items to the memory each time. Better to pass SKUs that you already have in the list of subscriptions and fetch which ones are onetimers

Reply:
https://github.com/softwareone-platform/mpt-extension-sdk/pull/63

Reply:
Done but requires SDK change (above) approval

Reply:
SDK v update